content/analyst/inbox-and-design-tracker/microsoft-snds-set-up-and-configuration.md

```diff
@@ -0,0 +1,128 @@
+
+1. **You must be marked as a Manager on your Inbox Tracker account.** If you are unsure whether you are a Manager, please contact our support team for guidance.
+2. **You have to be sending on dedicated IPs.** If you are unsure whether you are sending on shared or dedicated IPs.
+3. **You will need to know your ESP’s abuse/postmaster address** 
+
+	1. For example, for SparkPost, it’s [abuse@sparkpostmail.com](mailto:abuse@sparkpostmail.com)
+	2. If you are unsure of this, reach out to your ESP for more guidance
+4. **A role account on Outlook**. This is just an ordinary Outlook email address you set up to access to the SNDS dashboard. We recommend making this account accessible across your team/organization. For example, the role account could be *[yourorganization@outlook.com](mailto:yourorganization@outlook.com)* or *[crmteam@outlook.com](mailto:crmteam@outlook.com).*
+
+ Once all requirements have been met, you are ready to set up SNDS and link it to Inbox Tracker!
+
+---
+
+**Step 1 - Navigate to the Microsoft [Smart Network Data Service Page](https://postmaster.live.com/snds/index.aspx) and select Request Access.** 
+
+![](media/microsoft_snds_set_up_and_configuration/image_2.png)
+
+**Step 2 - While on the [Request Access](https://postmaster.live.com/snds/addnetwork.aspx) page, input your IP address or CIDR range into the *Network to request access for* input box and click *Submit***
+
+![](media/microsoft_snds_set_up_and_configuration/image_3.png)
+
+**Step 3 - After submitting your IP request you'll then be prompted with a list of verification [email addresses](https://postmaster.live.com/snds/FAQ.aspx#AddressChoosing) associated to the IP or IP ranges you entered.** Choose the valid email address that your service provider recommends submitting requests through.
+
+ Once the request has been submitted, your Pending Requests will appear on the Request Access page:
+
+![](media/microsoft_snds_set_up_and_configuration/image_4.png)
+
+*Please note, while you can finish setting up SNDS in Inbox Tracker before your request has been authorized, data will only start populating after authorization has been granted.* 
+
+**Step 4 - After the request has been submitted, you need to Enable Automated Access to get the feed links.** To do this, click the *View Data* link on the left.
+
+![](media/microsoft_snds_set_up_and_configuration/image_5.png)
+
+ Then, select *View or change your automated access settings*.
+
+![](media/microsoft_snds_set_up_and_configuration/image_6.png)
+
+ Lastly, on the automated access settings page, you'll need to click the *Enable Automated Access* button.
+
+![](media/microsoft_snds_set_up_and_configuration/image_7.png)
+
+**Step 5 - Once automated access has been enabled, you will see two links that for your automated SNDS data feed and your access key.** The first line (the one ending with "data.aspx") is the one you will need to integrate your SNDS data feed with Inbox Tracker. Copy this link for Step 6.
```

Comment:
`...you will see two links that for your automated SNDS data...` probably could drop `that`